03-path-application-development/302-app-discovery/readme.adoc

```diff
@@ -121,12 +123,12 @@ Wait for ~3 mins for the load balancer to accept request.
 
 Access the application using the following URLs with `curl` or via a browser:
 
-  http://<host>
-  http://<host>?greet=ho
-  http://<host>?id=1
-  http://<host>?greet=ho&id=1
+  http://<ingress>
```

Comment:
I think this should be left as `<host>`. We can add clarity around the instruction, such as running the `kubectl get svc/webapp-service` command first, then stating to use that value as `<host>`.